src/FeatureDescriptors.jl

```diff
@@ -54,6 +54,15 @@ function label(D::Type{<:Descriptor})
     return Symbol(s)
 end
 
+"""
+    parents(D::Descriptor) -> Vector
+
+Returns the parent [`Descriptor`](@ref) for each table names returned from
+[`sources`](@ref). Similar to `Base.parent`, we return input `D` when there
+is no parent for a source.
+"""
+parents(D::Type{<:Descriptor}) = fill(D, length(sources(D)))
```

Comment:
Is the idea that a descriptor is automatically its own parent unless specified otherwise?

Reply:
That's correct. If we didn't want to do this then we should make this release breaking as the new interface tests will breaking downstream packages.